demo/src/public/experiment-page2.js

```diff
@@ -1,14 +1,34 @@
 (function() {
     'use strict';
 
-    var lou = new Mozilla.TrafficCop({
+    var variants = {
+        'a': 30,
+        'b': 30,
+        'c': 30
+    };
+
+    function handleVariation(variation) {
+        // wait until DOM is ready to be manipulated...
+        domReady(function() {
+            // make sure variation is one we are expecting (and not noVariationCookieValue)
+            if (variants.hasOwnProperty(variation)) {
+                var target = document.getElementById('var-' + variation);
+                target.classList.remove('hidden');
+            }
+        });
+    }
+
+    // non-jquery document ready function
+    // http://beeker.io/jquery-document-ready-equivalent-vanilla-javascript
+    function domReady(callback) {
```

Comment:
Perhaps if we explicitly called out that callback experiments should not be in the head (for perf reasons) in both the readme and demos, this might be a good way to go and avoids needing to add dom ready code?